
Needed to build amd74xx.c which does also handle nvidia nforce chipsets (like
the one on my epox motherboard).  -- Clifford

--- ./include/linux/pci_ids.h.orig	2003-04-26 00:40:18.000000000 +0200
+++ ./include/linux/pci_ids.h	2003-04-26 00:40:31.000000000 +0200
@@ -1944,3 +1944,6 @@
 #define PCI_DEVICE_ID_MICROGATE_USC	0x0010
 #define PCI_DEVICE_ID_MICROGATE_SCC	0x0020
 #define PCI_DEVICE_ID_MICROGATE_SCA	0x0030
+
+#define PCI_DEVICE_ID_NVIDIA_NFORCE_IDE		0x01bc
+#define PCI_DEVICE_ID_NVIDIA_NFORCE2_IDE	0x0065
